Update LAN event (#538)
ZeusWPI/zeus.ugent.be: content/events/22-23/lan.md

location: Therminal, Hoveniersberg 24, 9000 Gent
description: Zeus' elfde LAN-Party!
header_text_background: true
banner: https://pics.zeus.gent/zTVVLv1sQAlbS1WFZG60zQQbk4eOjvWIPudB9Uhd.jpg
locationlink: $therminal
---

# Do you wanna play a game ?
Het is weer de tijd voor de jaarlijkste LAN party van Zeus WPI. Voor iedereen die tot in de laatste uurtjes gezamelijk wilt gamen.

## Info

**Datum**: Vrijdag 21 oktober - Zaterdag 22 oktober

> **Tijd**: xx:xx tot wanneer we niet meer kunnen

Author: ```suggestion
**Tijd**: 18:00 tot wanneer we niet meer kunnen
```
Op de - al geprinte - affiche staat 18u00 dus dat staat vast 😆

location: Therminal, Hoveniersberg 24, 9000 Gent
description: Zeus' elfde LAN-Party!
header_text_background: true
banner: https://pics.zeus.gent/zTVVLv1sQAlbS1WFZG60zQQbk4eOjvWIPudB9Uhd.jpg
locationlink: $therminal
---

# Do you wanna play a game ?

> Het is weer de tijd voor de jaarlijkste LAN party van Zeus WPI. Voor iedereen die tot in de laatste uurtjes gezamelijk wilt gamen.

Reviewer: ```suggestion
Het is weer tijd voor de jaarlijkste LAN-party van Zeus WPI. Voor iedereen die tot in de late uurtjes gezamenlijk wilt gamen.
```

## Info

**Datum**: Vrijdag 21 oktober - Zaterdag 22 oktober

**Tijd**: 18:00 tot wanneer we niet meer kunnen

**Locatie**: Therminal, Hoveniersberg 24, Gent

> **Meenemen**: Stekkerdoos + ethernetkabel

Reviewer: ```suggestion
**Meenemen**: Stekkerdoos + ethernetkabel + jouw pc
```

Maybe? Alhoewel er natuurlijk ook couchgaming gaat zijn (mss kunnen we daar zelfs nog iets over zeggen), maar voor de pc games moeten ze hun eigen pc voorzien (al is dat mss wel obvious als je je stekkerdoos en ethernetkabel moet meenemen)

Author: Doe pc/laptop?

Reviewer: ook goed 👍

> **Inschrijven**: Je moet je niet inschrijven voor de lan zelf. Info volgt i.v.m. competitions

Reviewer: ```suggestion
**Inschrijven**: Je moet je niet inschrijven voor de LAN zelf. Info volgt i.v.m. competities
```

## Competitions

Trackmania, Minecraft, Mario Kart, Nidhogg en nog veel meer!

## Prijzen

Er zijn in totaal honderden euro's aan prijzen, dus je hoeft niet eens de beste te zijn om met iets naar huis te gaan!

Tot dan!
Zeus WPI

> ![Lan party poster](https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg "Lan party poster")

Reviewer: ```suggestion
![Lan party poster](https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg "Lan party poster"){ width=60% }
```

assuming dat we pandoc gebruiken (https://stackoverflow.com/a/41169162/19735148), mss dat dit iets overzichtelijker gaat zijn.

Author: Ik denk dat dat nie gaat werken, volgens mij moet je met een `<img/>` werken op github

Reviewer: welja, maar de site wordt niet door GitHub gehost right? 't is enkel de preview hier dat niet zou werken dan bv. Maar dit was puur een poging, dus geen idee of dit op de site gaat werken

Author: Kzal eens dit proberen:
`<img src="https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg" max-width=700px />`

Author: Aaahja je hebt gelijk haha toch jouw ding proberen

Reviewer: > Kzal eens dit proberen:
> `<img src="https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg" max-width=700px />`

Moest je laatste commit niet werken, dan zou dit werken, alleen moet je dat in een `style` zetten, dus 
`<img src="https://pics.zeus.gent/zv92vxNreqJaj1cLg7M0kj7ODDYzvXhNta2bPCuF.jpg" style="max-width: 700px;" />`
